docs/source/dev-and-test/setup-run-node.md

```diff
@@ -38,3 +38,41 @@ The BigchainDB [CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/b
 Ian Worrall of [Encrypted Labs](http://www.encryptedlabs.com/) wrote a document (PDF) explaining how to set up a BigchainDB (Server) dev machine on [Cloud9](https://c9.io/):
 
 [Download that document from GitHub](https://github.com/bigchaindb/bigchaindb/raw/master/docs/source/_static/cloud9.pdf)
+
+
+## Option C: Using a Local Dev Machine and Docker
+
+You need to have recent versions of [docker engine](https://docs.docker.com/engine/installation/#installation)
+and [docker-compose](https://docs.docker.com/compose/install/), and possibly
+[docker-machine](https://docs.docker.com/machine/install-machine/),`if you are
+running on Mac OS X or Windows, for instance.
+
+Build the images:
+
+```bash
+$ docker-compose build
+```
+
+Start RethinkDB:
+
+```bash
+$ docker-compose up -d rdb
+```
+
+Start a BigchainDB node:
+
+```bash
+$ docker-compose up -d bdb
+```
+
```

Comment:
Can you add a paragraph about where to access the RethinkDB web interface (i.e. what URL) and the BigchainDB HTTP API?